src/common/Common.UI/Common.UI.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  
+  <!-- Look at Directory.Build.props in root for common stuff as well -->
+  <!-- This is used by the installer, $(SolutionDir) won't work here -->
+  <Import Project="..\..\Common.Dotnet.props" />
```

Comment:
e.g. we` can define property called "MainSolutionDir" with MSBuildThisFileDirectory\..\..\..

Reply:
Yeah.  That I think is going to be one of the last optimizations I may do.

Reply:
* in a different PR was my thought.  if you feel i should add it in, 100% happy to.  My thought is it is a easy self contained change